src/prefabs/structures/AutocompleteInput/options/validation.ts

```diff
@@ -0,0 +1,19 @@
+import { showIf, text, variable, toggle } from '@betty-blocks/component-sdk';
+
+import { showOn } from '../../../../utils';
+
+export const validation = {
+  required: toggle('Required'),
+  validationValueMissing: variable('Value required message', {
+    value: ['This field is required'],
+    ...showOn('required'),
+  }),
+  closeOnSelect: toggle('Close dropdown after select', { value: true }),
+  disabled: toggle('Disabled'),
```

Comment:
toggles should have a valid value, does this show up in the lint?

Reply:
I've added an explicit value, it looks like the `attrs` object on the toggle function is optional 
```typescript
export declare const toggle: (label: string, attrs?: Attributes) => OptionProducer;
```

Reply:
Ah one moment, I see disabled has been put down twice, my linter didn't catch that for some reason.

Reply:
> ```export declare const toggle: (label: string, attrs?: Attributes) => OptionProducer;```

I thought i'd made it explicit, my mistake.